src/main/res/values/strings.xml

```diff
@@ -447,7 +447,7 @@
     <!-- title shown above a list contacts where one should be selected (eg. when a webxdc attempts to send a message to a chat) -->
     <string name="send_message_to">Send Message to…</string>
     <string name="enable_realtime">Realtime Webxdc Channels</string>
-    <string name="enable_realtime_explain">Enable Webxdc realtime APIs to create direct connections between devices.</string>
+    <string name="enable_realtime_explain">Webxdc realtime APIs allow direct connections between devices. If enabled, chat partners may be able to see your IP address.</string>
```

Comment:
```suggestion
    <string name="enable_realtime_explain">This allows webxdc apps to create direct connections between devices. If enabled, chat partners may be able to see your IP address.</string>
```

Better not use the technical term "API here". 

I'm wondering if we should just add "If disabled, some apps may not work."

Reply:
i'd say it's clear that if you disable this option that realtime apps may not function.  
here is my suggestion (avoiding "webxdc realtime" which really does not tell users much if they don't know about it yet). 

```suggestion
    <string name="enable_realtime_explain">Enable real-time support for apps shared in chats. If enabled, chat partners may be able to discover your IP address after you started an app.</string>
```

Reply:
note, that this string is only the _subtitle_, the _option title_ itself still has "webxdc" and "realtime" in, i think, we need to change both (it is only github, that does not allow to adapt unchanged lines)

so, if we take the suggestion from @hpk42 , maybe change the title from `Realtime Webxdc Channels` to `Support Real-Time Apps`?